### PR TITLE
Move to Ruff

### PR DESCRIPTION
**Pull request by AntoinePrv**
Tired of modifying X files to configure Y tools.

### CODE REVIEW

**Comment by jonashaag** [comment]
Cool! I guess pre-commit should also be updated?

**Comment by AntoinePrv** [comment]
Working on fixing many warnings

**Review by jonashaag — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -2,3 +2,9 @@
 minversion = "6.0"
 tmp_path_retention_policy = "failed"
 addopts = "--color=yes"
+
+[tool.ruff]
+line-length = 88
+target-version = "py37"
```
[the remark this answers is not in the document]
Comment (jonashaag) [reply pyproject.toml:8]: OK, yes it is

**Comment by JohanMabille** [comment]
> modifying X files

x-files-paranormal

**Pull request merged**